src/Handlers/DetachAttachment.php

```diff
@@ -2,11 +2,29 @@
 
 namespace Froala\NovaFroalaField\Handlers;
 
-use Froala\NovaFroalaField\Models\Attachment;
 use Illuminate\Http\Request;
 
 class DetachAttachment
 {
+    /**
+     * The attachment model class name.
+     *
+     * @var string
+     * @psalm-var class-string<\Froala\NovaFroalaField\Models\Attachment>
+     */
+    protected $attachmentModelClassName;
+
+    /**
+     * Create a new class instance.
+     *
+     * @param  string $attachmentModelClassName
+     * @return void
+     */
+    public function __construct($attachmentModelClassName)
+    {
+        $this->attachmentModelClassName = $attachmentModelClassName;
```

Comment:
Let's get model class from config directly, without requirement to pass it into constructor.
Also let's omit "name" suffix and attachment prefix, we already in `DetachAttachment` class context. The repetition of context is mostly redundant.

Reply:
> Let's get model class from config directly, without requirement to pass it into constructor.

Why would you create a tight coupling between the handlers, which are small and single-purpose classes, and the config helper? There's no need for them to know about the framework and its helpers. Injecting the model class makes them less dependent.

> Also let's omit "name" suffix and attachment prefix, we already in DetachAttachment class context. The repetition of context is mostly redundant.

Why not be as explicit as possible? Is there a cost to having a slightly longer variable/property name?